README.md

```diff
@@ -26,9 +26,16 @@ Abaixo encontram-se guias para instalação em Windows e Linux.
 
 1. Fazer clone do repo:
 
-```bash
-git clone https://github.com/diogotcorreia/resumos-leic.git
-```
+- Por `https`:
+
+  ```bash
+  git clone https://github.com/diogotcorreia/resumos-leic.git
+  ```
+
+- Por `ssh`:
+  ```bash
+  git clone git@github.com:diogotcorreia/resumos-leic.git
+  ```
```

Comment:
I don't think this is needed, only contributors have write access, therefore HTTPS is always enough for everyone else